pyodide-build/pyodide_build/tests/test_buildall.py

```diff
@@ -22,12 +23,18 @@ def test_generate_dependency_graph():
     assert pkg_map["beautifulsoup4"].dependents == set()
 
 
-def test_generate_packages_json():
+def test_generate_packages_json(tmp_path):
+    # Set up directory to store dummy package files for SHA-256 hash verification
+    if not os.path.exists(tmp_path):
+        os.makedirs(tmp_path)
```

Comment:
```suggestion
    """Set up directory to store dummy package files for SHA-256 hash verification"""
```
I think it's already created by default, so you don't need to do this.
Otherwise, you since it's a pathlib object you can use `tmp_path.mkdir(exist_ok=True)`

Reply:
You're right, the directory is already created.